pubtools/pulplib/_impl/model/repository/file.py

```diff
@@ -13,6 +13,16 @@
 LOG = logging.getLogger("pubtools.pulplib")
 
 
+@attr.s(kw_only=True, frozen=True)
+class IsoImporter(Importer):
```

Comment:
Could we please name this `FileImporter` instead, to match `FileRepository`?

It's the same reasoning as for `FileRepository` itself.  Yes, technically in Pulp's code and type IDs it refers to it as "iso", but that's a mistake and it's actually a generic file handler.  We don't need to repeat the mistake in our own APIs, and by avoiding that we'll be better off if we ever want to port this code to Pulp3.